backend/forgather/src/main/java/com/forgather/domain/space/model/Space.java

```diff
@@ -60,4 +60,8 @@ public boolean equals(Object object) {
     public int hashCode() {
         return Objects.hashCode(id);
     }
+
+    public boolean isExpired() {
+        return expiredAt.isBefore(LocalDateTime.now());
+    }
```

Comment:
파라미터로 시간을 받아 비교하는 방식은 어떻게 생각하시나요??
```suggestion
    public boolean isExpired(LocalDateTime now) {
        return expiredAt.isBefore(now);
    }
```